webcomponents.js/webcomponents.js.d.ts

```diff
@@ -41,6 +46,10 @@ declare module "webcomponents.js" {
     export = webcomponents;
 }
 
+interface Element {
+    createShadowRoot(): webcomponents.ShadowRootPolyfill;
```

Comment:
should there also be a `shadowRoot` field here because the file you linked defines a getter for it? or is that already available on `Element`?